### PR TITLE
Recharts & stats

### PR DESCRIPTION
**Pull request by xsharonhe**
I will add a recommendation feature once users is complete.
Screen Shot 2021-01-03 at 11 21 00 PM
Screen Shot 2021-01-04 at 12 25 53 AM

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -17,11 +21,16 @@
     "@types/node": "^12.0.0",
     "@types/react": "^16.9.53",
     "@types/react-dom": "^16.9.8",
+    "@types/react-select": "^3.1.2",
     "axios": "^0.21.1",
+    "prop-types": "15.5.10",
```
Comment (xsharonhe) [inline frontend/package.json:26]: Reminder to myself to remove unused packages

File: frontend/src/scenes/Discover.tsx [hunk 2]
```diff
@@ -108,8 +109,8 @@ const Discover = () => {
               ) : (
                 <CardWrapper>
                   {loadRentals}
-                  {maxRange <= 50 && (
-                      <SButton onClick={loadMore} isInverted={false}>
+                  {maxRangeRentals <= 50 && (
```
Comment (xsharonhe) [inline frontend/src/scenes/Discover.tsx:112]: This should be less and I can make it so it's not fixed

**Review by xsharonhe — COMMENTED** [review]
Just for myself to remember tmrw haha also push your changes too esp the FAQ page

**Review by danielle-dsouza — APPROVED** [review]
Looks good to me

**Pull request merged**